website/cue/reference/base/configuration.cue

```diff
@@ -547,5 +549,224 @@ base: configuration: configuration: {
 			Secrets are loaded when Vector starts or if Vector receives a `SIGHUP` signal triggering its
 			configuration reload process.
 			"""
+		common:   false
+		required: false
+	}
+	acknowledgements: {
```

Comment:
Please note that these configuration are generated from the Rust original code. I don't port the config from `configuration.cue` to Rust docs.